src/components/Layout/Footer/index.tsx

```diff
@@ -26,131 +26,71 @@ const Footer: React.FC<Props> = ({ isWhiteBg, isContactPage }) => {
     );
 
   return (
```

Comment:
There is an issue with footer on other pages, please check that issue, we can have the same footer component(new footer design), but it needs to work on each page

Reply:
Okay, I corrected it.